ts/features/itwallet/issuance/store/reducers/index.ts

```diff
@@ -19,15 +28,35 @@ const reducer = (
   switch (action.type) {
     case getType(itwStoreIntegrityKeyTag):
       return {
+        ...state,
         integrityKeyTag: O.some(action.payload)
       };
     case getType(itwRemoveIntegrityKeyTag):
-    case getType(itwLifecycleStoresReset):
       return {
+        ...state,
         integrityKeyTag: O.none
       };
+    case getType(itwIntegritySetServiceIsReady):
+      return {
+        ...state,
+        integrityServiceReady: action.payload
+      };
+    case getType(itwLifecycleStoresReset):
+      return {
+        integrityKeyTag: O.none,
+        integrityServiceReady: undefined
+      };
```

Comment:
This should not be set to `undefined` here, ore the `itwLifecycleStoresReset` will remove the fleg right before the polling starts in the eid machine

```suggestion
    case getType(itwLifecycleStoresReset):
      return {
       ...state,
        integrityKeyTag: O.none
      };
```